### DISCUSSION TITLE
Protocol changed to ISAKMP

### DISCUSSION THREAD

**Discussion by BeomjunGoo** [ANLAB-KAIST/KENSv3#125] [Project Q&A 2. Transport Layer (TCP)]
Hi, 

For some of the packets that we've sent in our code, the protocol changes to ISAKMP for the respective ones. 

Is it okay to use as-is? or does this indicate that our implementation is incorrect?

Moreover, if so what are potential problems..?

Thanks

**Comment by whkim0** [comment]
I guess that you have opened your pcap file and found ISAKMP. Can you upload the packet along with the screen capture of Wireshark opening your packet?
